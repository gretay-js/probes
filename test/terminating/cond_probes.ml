let g x =
  Printf.printf "g:%d\n" (x-1)

let f x =
  if x > 0 then
    [%probe "foo" (g x)];
  Printf.printf "f:%d\n" x

let () =
  f (Sys.opaque_identity 10)
