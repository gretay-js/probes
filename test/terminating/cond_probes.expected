g:9
f:10
